.github/workflows/eden.yml

```diff
@@ -1,105 +1,26 @@
 ---
 name: Eden
-on:  # yamllint disable-line rule:truthy
-  pull_request:
-    branches: [master]
+
+# yamllint disable-line rule:truthy
+on:
+  push:
+    branches:
+      - "master"
+    paths-ignore:
+      - 'docs/**'
+  pull_request_review:
```

Comment:
We had run without approval before, well, it is debatable and probably not good for some kind of paid runners. But is it possible to implement workflow_dispatch for users with permissions to not wait for approval...

Reply:
We can actually restore previous permissions, but is it taking long time to wait for approval in eden repository?

Reply:
moved it back to the way it was